### PR TITLE
UCT/RC/DC: Add cyclic XRQ support for HWTM with DEVX

### PR DESCRIPTION
**Pull request by brminich**
## What
Add support for cyclic XRQ topology with DEVX and tag offload  

## Why ?
It gives ~10-20 nano performance improvement on latency benchmarks

## How ?
Setting wq type when creating XRQ via DEVX. To be added for SW TM as well

### CODE REVIEW

**Comment by brminich** [comment]
@Artemy-Mellanox,  can you plz review?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12068/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/e88c43f61942a8fd706dfbffab91e55b7767e933/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/04471e58982124f1ea115c8b0c98d70e8ee8e192/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/6f50567ae862510b083aa19a24c0c8198d1eb052/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/b8116172bc78eb26cf7646a380dc8d9b9e0468b4/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/ac0345f4da39e2fe7be504d868f2071e0cf6cb1f/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/1df99f119be04f58e3c245c5d36b48cddf4b4b06/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/f788e1a8a0f762322f1238fad9c69678793ebd93/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/2e33fb7d3cbc61948fbe6ab436aa35719bd3cc30/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/fdeada5705642a411fe19cacca9183138617460a/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/a800d23550e9632fafbc59bf166357d2032b7c2e/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/ac0cedd48f5f2303d6d640efdfe78ec04dd2664e/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/10589b65329750d20831e118917381661b077f38/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/3bff5aa8708485041f9b909dd7d81eced3251299/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/ed2076fd3fe06910c2cc3e8cc4306e9f6656ab72/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/65fd22a6387d4c8ca63d483abec4b12c3feffdf1/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/d5afabc83b6d94ea1878a8a74cc6dc40cdf67ef0/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/fd6357e32a1e46e7d5a39ca7c9d49a102da5dcc9/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/08cbdd14935ae57794dfecda94f82f1d850abe17/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/72a168240ea672057a7c32b430f72f32a9bb3d5d/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/d1fb8320a4b0b6951d50937910431118f1835a06/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/f146b1b9ef43dcda0f188073866293cc22ff946f/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/66fda6270e775aac6c348d9da0618861fe3a77e3/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/b5b62301ef7a8d28ec6da46ba3830ba3da9e88f1/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/d10196c556b58c20c9f953fb994c60d080cc7e09/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/3b2e4defc3e7b8944112a79242a4698c7edc5a4c/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/7e26e5983276762fc7c4981590acffeb87b7b1d0/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/2fd8675edd74f5144470c7b44070831e9c2a01ec/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/9ea48689e7981eb41ccd466cb0e5eac9edb793e7/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/87cd9d816723844b829036634fd46917/raw/b3b2f3b3e1201cd9cea15aacf4c71f1bef616587/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12078/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/fb4ffe2450925ef659a10fdcabf42f981a659ae2/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/a76ea97775f309c8e4bed707a04eee089fb368e4/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/fc1307f8772324480ed029a555cef6c2c2d6075c/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/0ef024602c1384f85117f74055bcb757db4918a9/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/afb04cd5e86b807fcb087e61ee3ccf20a9c85783/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/a586e2808f69602d4a457e32ab5ac608102dfbb2/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/7c71b1ee45c0f75f3ddb904923fced7cf2900b88/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/26231663b5cf97d41428137e849bca539b0ae1cc/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/8f0d35d660ae6afcf6a0bb2acc06700e7acbcb60/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/78d94acb7a913380e91283d45ec5a5f54a1621a3/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/4d666c13a9102f53fca63db400090e871f3ecfb5/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/93ac20a45f7367f04a3dabc4084d82f0ab488e8c/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/3a18bd12dfabd59c49c9faced1772a8afcf632ac/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/70c919b52128003bff3ae22d4bbf3431a965199e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/ab6d3c86b51aa64215bef865c17fef833bfe27bf/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/15124e39099f6d9da15ae33b209d30c9f201fb47/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/633f627db18412f7c78aa99efcd2528c78fd0285/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/c4675419af3c5b00dbedb0b4fececd549e17424d/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/c5020a5ca3e9d4f06760f32ec25c4d262f763141/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/bd86115f6c16bfc2d28facb39cf7b107c07b2b8e/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/7fd69eca3a59565eb3de608db74144accd89365a/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/7d403933bfe9df3d447bea7fab0b8d4ac9145737/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/a565ea9928bf7f1a807ac72d53f658c039d7a0f3/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/756fa7aebfe6ef04d75c186f8053c4e9ce5ab8f1/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/b71301eef514d87f78316c1fa41bbe33f2719604/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/d4dcb35ddd3b987de85ba7e06a4b10e8fa3d8855/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/7cbef82569dd91d4de6ad76d94fa0a294f1c6a92/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/28a3a7786e4552f23eef37ac8a3ad38fc0f284e0/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b71ce9a43283837ab0567f2a2b14ff80/raw/a148eb8a398328f0e82093b5caf5311d72560805/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
@yosefe, plz review

File: src/uct/ib/rc/accel/rc_mlx5_common.c [hunk 1]
```diff
@@ -46,6 +57,18 @@ ucs_config_field_t uct_rc_mlx5_common_config_table[] = {
    ucs_offsetof(uct_rc_mlx5_iface_common_config_t, exp_backoff),
    UCS_CONFIG_TYPE_UINT},
 
+#if HAVE_DEVX
+  {"SRQ_TOPO", "auto",
+   "SRQ topology type. The types are:\n"
+   "\n"
+   "list       SRQ is organized as a buffer containing linked list of WQEs\n"
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_common.c]: need doc for auto?

File: src/uct/ib/rc/accel/rc_mlx5_iface.c [hunk 2]
```diff
@@ -447,6 +447,21 @@ static void uct_rc_mlx5_iface_event_cq(uct_ib_iface_t *ib_iface,
...
 
+static int uct_rc_mlx5_iface_srq_topo(uct_rc_mlx5_iface_common_t *iface,
+                                      uct_md_h md,
+                                      uct_rc_mlx5_iface_common_config_t *mlx5_config)
+{
+    /* Cyclic SRQ is supported with HW TM and DEVX only. */
+    if (((mlx5_config->srq_topo == UCT_RC_MLX5_SRQ_TOPO_AUTO) ||
+        (mlx5_config->srq_topo == UCT_RC_MLX5_SRQ_TOPO_CYCLIC)) &&
+        UCT_RC_MLX5_TM_ENABLED(iface) &&
+        ucs_derived_of(md, uct_ib_mlx5_md_t)->flags & UCT_IB_MLX5_MD_FLAG_DEVX) {
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_iface.c]: minor: add local variable of type uct_ib_mlx5_md_t*

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12082/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/e34662e7cf651f17e929f16c9508c4e56f2edd7c/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/4d9ae7a8b4691de6fa019476787ea3c3b9218c85/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/2d81b6fd9cc9452d39806c1db0f485c8e222f146/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/17985a64939e0daf9f931dce74acc3962a53f5a2/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/b2720692bd8bfd27c261847efa2ab5705c1f9073/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/1eabab60bb6a8c8358cc195063edb816da8d3a78/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/eb0adcfea909a427bfce90091cc1ed5528aa8f2d/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/9c8e8cc6b1ab4991820ab99d30504d3b5ac2b6d0/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/026adde65dd29ff72f153210591054c90184a973/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/0c80382e0a8c785755303aee7ebd9fa2e4748c57/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/7b11fc7cf77cb55de363dd5221d831d7f2f3974d/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/6dbaef5ffaeff81234ca1f2dbecf4dbc1b293292/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/9295c3026f76d8dcfb1bff36f7c9cc265269ebc1/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/5ef38c7494b352080acb38acefc18b3a282dd95c/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/71ca7b7f88e3270943adbda729a0aca5f7bc61b4/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/8197048d24c2f95d2f8c768abe1054a027087ca9/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/4e6a14c8f986ce4a16374a287d0a4fef13a478bc/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/85b1e0e6f48e81dd46ff2a3ff849d7234459bd34/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/5bd6b48133c50b00f686044d38c3acccb692abd5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/805b9f80c62db73b0b6baf2e8cfb9bd9e0126746/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/d374c8fe5221108380cadb4e2c685a27b2295a8e/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/098d7e581c8785b72018569a728f4a840811cf0d/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/2934196823da3dfdae6c3536b4545da73b17ad3c/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/ca52d191ebf37dacb25581252805e29d50838a18/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/b0ed6f24dca66b67442491e373232cc46eac8628/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/15059921430c6d2939f90b207d782ba7f49e3859/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/b7b3f736a6a27163b4baea559cd4ff3b2d4a2d3b/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/6028ece230b1a07c3f3a7a64bf1d6afe1be0cf32/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/7d70d4204824987438de6787537f7329/raw/53ee055d4c3a52df44e1490ed2c003bab30765c0/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
@yosefe, ok to merge?

**Pull request merged**